update/ansible/playbook/tasks/roles/clickhouse/tasks/main.yml

```diff
@@ -33,6 +33,8 @@
       - percona-clickhouse-server
       - percona-clickhouse-common-static
       - percona-clickhouse-server-common
+      - clickhouse-client-21.3.20.1-2
```

Comment:
What about uninstalling versions prior to 21.3.20? Or was this the only one used in PMM?

Reply:
I think a few lines above does that trick.